packages/runner-sdk/lib/sync.ts

```diff
@@ -80,7 +81,7 @@ export abstract class NangoSyncBase extends NangoActionBase {
             return results;
         }
         return results.map((result) => {
-            if (result && '_nango_metadata' in result) {
+            if (result && typeof result === 'object' && '_nango_metadata' in result) {
```

Comment:
weren't types supposed to prevent the need for excessive defensive programming? 🙃